### PR TITLE
feat(about): added some styling to about page and read from `README.md`

### PR DESCRIPTION
**Pull request by peipacut**
Only managed to get as far as left aligning the text and having it read from the project readme. I just want to ask:
- should the page by affected by light/dark mode (currently this is what is happening when dark mode is enabled) 
image
- which font should we use (currently using Roboto), does this matter?

### CODE REVIEW

**Comment by LorenzoValentine** [comment]
> Only managed to get as far as left aligning the text and having it read from the project readme. I just want to ask:
> 
> * should the page by affected by light/dark mode (currently this is what is happening when dark mode is enabled)
>   image
> * which font should we use (currently using Roboto), does this matter?

All the fonts from [Google Fonts](https://fonts.google.com/) are free or open source. They can be used for commercials.

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]
Needs a merge from the current doc into README. and move the devemlopment part in README to another file.

**Comment by peipacut** [comment]
all the development details should be in `docs/gettingstarted.md`. the about information is now in the `README`

File: README.md [hunk 1]
```diff
@@ -7,51 +7,45 @@ previously *Physics in the Browser*
...
-3. Build the application
+## About the team
 
-   ```sh
-   npm run build
-   ```
+This project was developed in collaboration with team from ANU TechLauncher, a student industry collaboration program at the Australian National University.
 
-   The built application will be in the `dist` folder.
+The team is led by students of ANU college of Engineering, Computing and Cybernetics. Students with different levels of expertise in coding, design, content creation, and project management actively collaborate to create this website that presents a user-friendly simulation platform to the public.
```
Comment (Lutra-Fs) [inline README.md:39]: Maybe add a section about contributor? nothing else needs to change.

**Review by Lutra-Fs — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (peipacut) [reply README.md:39]: sorted

**Review by Lutra-Fs — APPROVED** [review]
LGTM! Let's wait for Lorenzo to review the changes. He is doing the RWD work

**Review by LorenzoValentine — APPROVED** [review]

**Pull request merged**